sdk/core/core-lro/package.json

```diff
@@ -33,6 +33,7 @@
   ],
   "main": "./dist/commonjs/index.js",
   "types": "./dist/commonjs/index.d.ts",
+  "browser": "./dist/browser/index.js",
```

Comment:
there seems to be an existing browser key value below that overrides this one: `os: false, process: false`

Reply:
yeah I think that one got left over from before the migration and is safe to remove (@mpodwysocki can you confirm?)

Reply:
There are no pieces that use `process` nor `os` in our browser bundles